release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -87,6 +87,14 @@ With {product-title} {product-version}, running `oc describe` on an image now re
 [id="ocp-4-13-networking"]
 === Networking
 
+[id="ocp-4-13-nw-metallb-ipaddresspool-assignment"]
+==== Enhanced MetalLB IPAddressPool for reducing its scope to specific namespaces and services
+
+With this update, you can assign IP addresses from a MetalLB `IPAddressPool` resource to services, namespaces or both. This is useful in a muti-tenant, bare-metal environment that requires MetalLB to pin IP addresses from an IP address pool to specific services and namespaces. You can assign IP addresses from many IP address pools to services and namespaces. You can then define the prioritization for these IP address pools so that MetalLB assigns IP addresses starting from the higher priority IP address pool.
```

Comment:
It seems that we use a comma before _or_ in a list the includes _or both_. I'm not sure it is an absolute grammar rule.